### DISCUSSION TITLE
OCR language user setting

### DISCUSSION THREAD

**Discussion by thndrbck** [ciur/papermerge#609] [Ideas]
On the OCR documentation page you ask for opinion on how the default OCR feature works. Thank you for adding the "no OCR' option for an uploaded document.
I liked the feature in the earlier version of Papermerge where a user profile had the default OCR language as a setting parameter.
But I also like that the OCR language can be applied to the document being uploaded, although personally I will rarely use this feature.

Thank you very much for a great product!!!

**Comment by ciur** [comment]
User settings, i.e. settings which user can change via UI/REST API, will be brought back soon.
